src/ragas/llms/base.py

```diff
@@ -294,5 +297,17 @@ def llm_factory(
     timeout = None
     if run_config is not None:
         timeout = run_config.timeout
-    openai_model = ChatOpenAI(model=model, timeout=timeout)
+
+    default_headers = None
+    base_url = None
```

Comment:
I think everything looks good - I'll push these as default arguments of the function to keep things consistent and fix the CI stuff